domains/lambert.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "lambert76",
```

your github username
```suggestion
    "username": "HONGLK",
```